### ISSUE TITLE
Error during the attempt the finding the element

### ISSUE THREAD

**Issue by igorzip228** [selenide/selenide-appium#54]
During the test execution the error was observed:

Apr 23, 2021 7:05:44 PM io.appium.java_client.remote.AppiumCommandExecutor$1 lambda$0
INFO: Detected dialect: W3C
SLF4J: Class path contains multiple SLF4J bindings.
SLF4J: Found binding in [jar:file:/Users/izahorodniy/.m2/repository/org/slf4j/slf4j-simple/1.7.30/slf4j-simple-1.7.30.jar!/org/slf4j/impl/StaticLoggerBinder.class]
SLF4J: Found binding in [jar:file:/Users/izahorodniy/.m2/repository/ch/qos/logback/logback-classic/1.2.3/logback-classic-1.2.3.jar!/org/slf4j/impl/StaticLoggerBinder.class]
SLF4J: See http://www.slf4j.org/codes.html#multiple_bindings for an explanation.
SLF4J: Actual binding is of type [org.slf4j.impl.SimpleLoggerFactory]
[main] INFO com.codeborne.selenide.impl.Plugins - Using implementation of com.codeborne.selenide.impl.PageSourceExtractor: com.codeborne.selenide.appium.AppiumScreenSourceExtractor
[main] INFO com.codeborne.selenide.impl.Plugins - Using implementation of com.codeborne.selenide.impl.PageObjectFactory: com.codeborne.selenide.appium.SelenideAppiumPageFactory
[main] INFO com.codeborne.selenide.impl.Plugins - Using implementation of com.codeborne.selenide.impl.ElementDescriber: com.codeborne.selenide.appium.AppiumElementDescriber
[main] INFO com.codeborne.selenide.impl.Plugins - Using implementation of com.codeborne.selenide.commands.Commands: com.codeborne.selenide.appium.commands.SelenideAppiumCommands
[main] INFO com.codeborne.selenide.appium.AppiumElementDescriber - Failed to get attribute class: org.openqa.selenium.WebDriverException: An unknown server-side error occurred while processing the command. Original error: The attribute 'class' is unknown. Valid attribute names are: (
    UID,
    accessibilityContainer,
    accessible,
    enabled,
    frame,
    label,
    name,
    rect,
    selected,
    type,
    value,
    visible,
    wdAccessibilityContainer,
    wdAccessible,
    wdEnabled,
    wdFrame,
    wdLabel,
    wdName,
    wdRect,
    wdSelected,
    wdType,
    wdUID,
    wdValue,
    wdVisible
)
Build info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:17:03'
System info: host: 'Ihors-MacBook-Pro.local', ip: 'fe80:0:0:0:c19:86b9:b9d4:a33b%en0', os.name: 'Mac OS X', os.arch: 'x86_64', os.version: '10.15.7', java.version: '11.0.4'
Driver info: io.appium.java_client.AppiumDriver
Capabilities {app: /Users/izahorodniy/IdeaProj..., automationName: XCuiTest, browserName: , bundleId: com.profusa.glucose.profess..., databaseEnabled: false, deviceName: Provectus's iPhone, javascriptEnabled: true, locationContextEnabled: false, networkConnectionEnabled: false, noReset: true, platform: MAC, platformName: IOS, platformVersion: 13.6.1, takesScreenshot: true, udid: 00008020-000E35E40112002E, webStorageEnabled: false}
Session ID: a8211637-f984-4787-accc-603b8c6636c3

**Comment by asolntsev** [comment]
@igorzip228 The message you provided is just an "**INFO**" log. It's not an error. 

_Are you really getting some error?_

**Comment by BorisOsipov** [comment]
@asolntsev it is from telegram chat. Original error was
```
Element should be visible {By.xpath: //XCUIElementTypeImage[@name='bluetoothDisconnected']}
Element: '<? class="?" enabled="true" name="bluetoothDisconnected">org.openqa.selenium.WebDriverException: An unknown server-side error occurred while processing the command. Original error: The attribute 'text' is unknown. Valid attribute names are: (
    UID,
    accessibilityContainer,
    accessible,
    enabled,
    frame,
    label,
    name,
    rect,
    selected,
    type,
    value,
    visible,
    wdAccessibilityContainer,
    wdAccessible,
    wdEnabled,
    wdFrame,
    wdLabel,
    wdName,
    wdRect,
    wdSelected,
    wdType,
    wdUID,
    wdValue,
    wdVisible
)
```

AppiumElementDescriber should not spam error message with weird stacktrace

**Comment by asolntsev** [comment]
@igorzip228 @BorisOsipov Please help, how can I get in Appium iOS:

1. Element text (In Android, I use `element.getAttribute("text")`)
2. Element tag name (in android, I use `element.getAttribute("class")`)

**Comment by asolntsev** [comment]
People in internet say that text is often contained in `label`, sometimes in `value` or `name`.

**Comment by asolntsev** [comment]
@igorzip228 @BorisOsipov Released `selenide-appium:1.6.4`. Waiting for your feedback. Does the error message look better now?
